src/components/articleView.jsx

```diff
@@ -67,6 +83,18 @@ const ArticleView = () => {
           </div>
         </div>
       </div>
+
+      <div className="container mt-4 mx-auto">
+        <h2>Comments associated with an article</h2>
+        {comments.map((comment) => (
```

Comment:
If there are no comments and the comments array is empty, can we have some sort of message for the user to tell them this?